WebApiCoreSeed.UnitTests/Services/UserServiceTests.cs

```diff
@@ -0,0 +1,59 @@
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.EF;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.Domain.Services;
+using Xunit;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    public class UserServiceTests
+    {
+        [Fact]
+        public async Task GetByIdAsync_ShouldGetUserById_WhenUserIdExists()
+        {
+            // Arrange
+            var optionsBuilder = new DbContextOptionsBuilder<WebApiCoreSeedContext>();
+            optionsBuilder.UseInMemoryDatabase();

```

Comment:
As talked on skype I'm leaving it so every test method is self-sufficient. Thanks!